### PR TITLE
Update NER and Nominatim READMEs, and update PR template

### LINKED ISSUE #761: nominatim preprocessor requires a README
Describe what it is, what it depends on, limitations, etc.

### PR DESCRIPTION
**Pull request by JRegimbal**
Resolves #761. This adds brief READMEs for the Nominatim and NER preprocessors, which previously lacked them. The PR template is also updated to include a checkbox for adding a README, so hopefully PRs like this one do not need to happen in the future!

---

## Required Information

- [x] I referenced the issue addressed in this PR.
- [x] I described the changes made and how these address the issue.
- ~[ ] I described how I tested these changes.~ (N/A)

## Coding/Commit Requirements

* [ ] I followed applicable coding standards (e.g., [PEP8](https://pep8.org/))
* [ ] I have not committed any models or other large files.

## New Component Checklist (**mandatory** for new microservices)

* [ ] I added an entry to `docker-compose.yml` and `build.yml`.
* [ ] I created A CI workflow under `.github/workflows`.

### CODE REVIEW

File: preprocessors/ner/README.md
Comment (jeffbl) [inline preprocessors/ner/README.md:1]: To fully understand this, especially for someone not familiar with the project, can we put in a specific example photo and its output?

**Review by jeffbl — CHANGES_REQUESTED** [review]
nominatim looks good, just one comment on ner

File: preprocessors/ner/README.md
Comment (JRegimbal) [reply preprocessors/ner/README.md:1]: Added!

**Review by jeffbl — APPROVED** [review]
Looks good. Thanks for adding example!

**Pull request merged**